### ISSUE TITLE
1.1: TOC is borked

### ISSUE THREAD

**Issue by VladimirAlexiev** [opengeospatial/ogc-geosparql#507]
https://docs.ogc.org/is/22-047r1/22-047r1.html
The TOC pane (left side) is broken: major headings are interspersed with minor headings in a random way (eg B should not include A.4 to A.7):
![image](https://github.com/opengeospatial/ogc-geosparql/assets/536250/6322aeeb-c4fa-4fd6-8293-8e6f7a9b84a8)

**Comment by ghobona** [comment]
@VladimirAlexiev Thank you for reporting this. We will review the source files to determine how the TOC can be fixed.

**Comment by situx** [comment]
Hi @VladimirAlexiev @ghobona, We have already raised some issues with @ogcscotts about the official publication not resembling any of the versions we build with the Continuous Integration of this repository.
After one iteration we are still not there and I have been working on consolidating the branches for a new try possibly tomorrow.
So, @VladimirAlexiev, if you do not see those issues in the GeoSPARQL 1.2 and GeoSPARQL Development Draft autobuilds on the Github page, OGC should hopefully correct them soon.

**Comment by VladimirAlexiev** [comment]
https://opengeospatial.github.io/ogc-geosparql/geosparql11/spec.html doesn't have this problem.
But it looks very different from (and less official than) https://docs.ogc.org/is/22-047r1/22-047r1.html.

Also, some of the headstone info is different.
The former says `Contributors: Paul J. Cripps, Krzysztof Janowicz`
but the latter lists them as `Editors` together with everyone else.

**Comment by situx** [comment]
Correct. 
What probably happened was:
1. OGC did not use the geosparql11 branch for publishing but the master branch which contains the version formatted for ISO publication, which is also where the correct publishing information was added
2. They built the document themselves and something happened that messed up the TOC and other things
3. Since it was already published we decided that it was ok to run along with the ISO-formatted standard
4. Last week I updated the master branch and GeoSPARQL12 branch to be consistent with latest changes we pushed into GeoSPARQL1.1 so that OGC can publish the master branch contents (hopefully correctly) as 1.1
5. Next step for us is then to merge new contents into master and use the Geosparql12 branch (which is ISO formatted) for bug fixes and revisions for 1.1
